From: Arjan van de Ven <arjan@linux.intel.com>
Subject: Silence acer wmi driver on non-acer machines
Date: Fri, 23 Jan 2009
Patch-mainline: Not yet

Small fix changing error msg to info msg in acer wmi driver

Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
---
 drivers/platform/x86/acer-wmi.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/platform/x86/acer-wmi.c
+++ b/drivers/platform/x86/acer-wmi.c
@@ -1310,7 +1310,7 @@ static int __init acer_wmi_init(void)
 		AMW0_find_mailled();
 
 	if (!interface) {
-		printk(ACER_ERR "No or unsupported WMI interface, unable to "
+		printk(ACER_INFO "No or unsupported WMI interface, unable to "
 				"load\n");
 		return -ENODEV;
 	}
